dog.unix.nullobsi.cantata.yml

```diff
@@ -0,0 +1,25 @@
+id: dog.unix.nullobsi.cantata
```

Comment:
`nullobsi.unix.dog` doesn't exist

Reply:
Since I own the domain unix.dog, I suppose dog.unix.cantata would be a better ID, so I'll update the references in the main repo as well.